muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/workspace/workspaceHelp/help/index.tsx

```diff
@@ -732,11 +763,13 @@ class Help extends React.Component<HelpMaterialsProps, HelpMaterialsState> {
 function mapStateToProps(state: StateType) {
   return {
     i18n: state.i18n,
+    status: state.status,
     workspace: state.workspaces.currentWorkspace,
     materials: state.workspaces.currentHelp,
     activeNodeId: state.workspaces.currentMaterialsActiveNodeId,
     workspaceEditMode: state.workspaces.editMode,
     isLoggedIn: state.status.loggedIn,
+    isStudent: state.status.loggedIn && state.status.isStudent,
```

Comment:
what's the point of checking if the user is logged in? 

isStudent is false if you are not logged in
As far as I see it, you always check if the user is logged in anyway every time you check if user is a student